replica/table.cc

```diff
@@ -2503,8 +2503,9 @@ std::vector<view_ptr> table::affected_views(shared_ptr<db::view::view_update_gen
 }
 
 static size_t memory_usage_of(const utils::chunked_vector<frozen_mutation_and_schema>& ms) {
-    // Overhead of sending a view mutation, in terms of data structures used by the storage_proxy.
-    constexpr size_t base_overhead_bytes = 256;
+    // Overhead of sending a view mutation, in terms of data structures used by the storage_proxy, as well as possible background tasks
+    // allocated for a remote view update.
+    constexpr size_t base_overhead_bytes = 2560;
```

Comment:
This is a super-important fix, that "Fixes" the pre-existing problem for which I opened a new issue https://github.com/scylladb/scylladb/issues/17364.
I tend to think this patch should go in a separate PR - in particular **this** patch should be backported far and wide, but more importantly, I think we need a better understanding and justification for this number 2560. Can you try say something in this comment the sum of which data structures comprise this sum? Could we use sizeof() of a bunch of structures or something like that to calculate this number? 
And could we REDUCE this number? (this can be left to later, as a new issue, but we need to discuss this question).

Reply:
I extracted the fix with a longer explanation to another PR(https://github.com/scylladb/scylladb/pull/17420)

There are a few things we could sizeof() but it would be difficult to get a total value.

It's not obvious (but likely possible) to predict which background tasks/continuations will be present while waiting for a reply to a remote view update, but I'm not sure how we can the memory they're using.
Also, if the code gets changed, we will need to find the things to sizeof() again, so this wouldn't really help with automation of the overhead estimation

Reply:
Thanks, I merged that other PR so you can remove this patch from this PR.